packages/select/src/Select.js

```diff
@@ -5,9 +5,11 @@ import SelectBase from './SelectBase'
 const Select = ({
   list,
   children,
+  placeholder,
   ...props
 }) => (
-  <SelectBase {...props}>
+  <SelectBase {...props} defaultValue={placeholder && ''} placeholder={placeholder}>
```

Comment:
I am concerned about the defaultValue. If the the Select is being used as a controlled component this may produce either an error or a warning. We can probably do without the defaultValue and leave it up to whoever uses this component to be responsible of applying defaultValue or value.

Reply:
That's a good point. You're probably right. I'll play around with it tomorrow maybe.

Reply:
Good thoughts @pixelfuture